src/main/java/at/hannibal2/skyhanni/utils/tracker/SkyHanniBucketedItemTracker.kt

```diff
@@ -36,7 +36,7 @@ class SkyHanniBucketedItemTracker<E : Enum<E>, BucketedData : BucketedItemTracke
         addItem(bucket, SKYBLOCK_COIN, coins)
     }
 
-    fun addItem(bucket: E, internalName: NEUInternalName, amount: Int) {
+    fun addItem(bucket: E, internalName: NEUInternalName, amount: Int, manuallyAdded: Boolean = false) {
```

Comment:
as mentioned before, i recomment moving this into skyhanni item tracker, avoiding the code duplication alltogether